pkg/services/live/live.go

```diff
@@ -398,8 +398,8 @@ type GrafanaLive struct {
 }
 
 func (g *GrafanaLive) getStreamPlugin(pluginID string) (backend.StreamHandler, error) {
-	plugin := g.pluginStore.Plugin(pluginID)
-	if plugin == nil {
+	plugin, exists := g.pluginStore.Plugin(context.TODO(), pluginID)
+	if !exists {
 		return nil, fmt.Errorf("plugin not found: %s", pluginID)
 	}
 	if plugin.SupportsStreaming() {
```

Comment:
Due to the change in model returned here, it would be great if @grafana/grafana-edge-squad could verify Live is still happy ❤️